core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -137,6 +125,37 @@ public final String getPath() {
     return path;
   }
 
+  /**
+   * Configure Application MetricReport instances.
+   */
+  private List<MetricsReporter> configureMetricsReporters(T appConfig) {
```

Comment:
Not specific to this method, as I see many methods follow this pattern, but why do you need to receive `appConfig` here? Why can't you just use `config`? And why does it need to be a `T`, instead of a `RestConfig`?

Reply:
Mostly following convention here. Since there isn't really anything application specific in this method I opted to pass the config in and mark it private rather than accessing Application.config. Leaving as is for now.